src/include/log.h

```diff
@@ -365,10 +365,9 @@ struct __wt_log_desc {
 #define WT_LOG_VERSION_SYSTEM 2
 
 /*
- * WiredTiger release version where log format version changed.
+ * WiredTiger release version where log format version changed. FIXME WT-8681 - According to
```

Comment:
nit: The FIXME should start on a new line

Reply:
Alas, `s_all` disagrees :-(

Reply:
hmm. Bit surprising. I think there's two way to fix that:
1. Add a second newline
```
* WiredTiger release version where log format version changed. 
* 
* FIXME WT-8681 - A
```

2. Add some leading whitespace
```
* WiredTiger release version where log format version changed. 
*   FIXME WT-8681 - A
```
Can we see if one of those fixes things?

Reply:
The first one worked, so I've gone with that (3ea7742). Out of curiosity, why the need to have it on a new line?

Reply:
For me it's just easier to see there's a fixme present in the code when reading. 
Purely cosmetic, hence the nit.